doc/concepts/cell.rst

```diff
@@ -20,90 +20,148 @@ Cells interact with each other via spike exchange and gap junctions.
 
-1. **Sources**
-2. **Targets**
-3. **Gap Junction Sites**
+A cell can have multiple sources, targets and gap junction site objects. Each object is ordered relative to other
+objects of the same type on that cell. The unique :gen:`cell_member` (:gen:`gid`, :gen:`index`) identifies an object
+according to the :gen:`gid` of the cell it is placed on, and its :gen:`index` on the cell enumerated according to the
+order of insertion on the cell relative to other objects of the same type.
 
-Cells interact with other cells via :ref:`connections <modelconnections>` or
-:ref:`gap junctions <modelgapjunctions>`. Connections are formed from **sources** to **targets**. Gap junctions
-are formed between two **gap junction sites**.
+The :gen:`gid` of a cell is used to determine its cell :ref:`kind <modelcellkind>` and
+:ref:`description <modelcelldesc>` in the :ref:`recipe <modelrecipe>`. The :gen:`cell_member` of a source,
+target or gap junction site is used to form :ref:`connections <modelconnections>` and
+:ref:`gap junctions <modelgapjunctions>` in the recipe.
 
-A cell can have multiple sources, targets and gap junction site objects. Each object has a local :gen:`index`
-relative to other objects of the same type on that cell.
-A unique (:gen:`gid`, :gen:`index`) pair defined by a :gen:`cell_member` can be used to uniquely identify
-objects on a cell in a global model.
 
 
-.. _model_cell_kinds:
+.. _modelcellkind:
 
-Cell kinds
-----------
+Cell kind
+---------
 
 .. table:: The 4 types of cell supported by Arbor
 
     ========================  ===========================================================
     Cell Kind                 Description
     ========================  ===========================================================
-    **cable**                 Cell with morphology described by branching
-                              1D cable segments and user configurable mechanisms.
-    **lif**                   Single-compartment no-mechanism leaky integrate-and-fire
-                              neuron.
-    **spiking**               Proxy cell that generates spikes from a user-supplied
-                              time sequence.
-    **benchmark**             Proxy cell used for benchmarking (developer use only).
+    **Cable cell**            Cell with morphology and user configurable dynamics.
+    **LIF cell**              Leaky integrate-and-fire neuron.
+    **Spiking cell**          Proxy cell that generates spikes.
+    **Benchmark cell**        Proxy cell used for benchmarking (developer use only).
     ========================  ===========================================================
 
-.. _model_cable_cell:
+.. _modelcablecell:
 1. **Cable Cells**
 
-   Cable cells are morphologically-detailed cells represented as branching linear 1D segments. They can be coupled
-   to other cell types via the following mechanisms:
+   Cable cells are morphologically-detailed cells. They can be coupled to other cells via the following
```

Comment:
Is mentioning somewhere 'branched linear 1D segments' not a good idea?

Reply:
It's mentioned in the cell description below. I thought it fit in better there. But we can mention it here too, if you think it's a good idea.

Reply:
I see it now, OK!